trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -8403,7 +8426,7 @@ subroutine readLinkSL( GNLINKSL,NLINKSL,route_link_f, route_lake_f, maxorder, &
                             tmpCHLAT(i),    tmpZELEV(i),     tmpTYPEL(i),   tmpORDER(i),    &
                             tmpQLINK(i,1),  tmpMUSK(i),      tmpMUSX(i),    tmpCHANLEN(i),  &
                             tmpMannN(i),    tmpSo(i),        tmpChSSlp(i),  tmpBw(i),       &
-                            tmpTw(i),       tmpTw_CC(i),     tmpn_CC(i)
+                            tmpTw(i),       tmpTw_CC(i),     tmpn_CC(i),    tmpChannK(i)
```

Comment:
This line reads channel loss from a table manually. I don't think this was tested, as the NWM uses a RouteLink.nc file.